atlasdb-commons/src/test/java/com/palantir/remoting/HeaderAccessUtilsTest.java

```diff
@@ -97,6 +101,15 @@ public void caseInsensitiveGetShortcircuits() {
         assertEquals(VALUE_2, HeaderAccessUtils.shortcircuitingCaseInsensitiveGet(testMap, KEY_2.toUpperCase()));
     }
 
+    @Test
```

Comment:
Shouldn't need this test, we can remove this method. We can probably drop the javax.ws.rs-api import as well, since I think this is the only place it's used.